### PR TITLE
Update changelog for tools v21.08.1 release [skip CI]

### PR DESCRIPTION
**Pull request by NvTimLiu**
Update changelog for tools v21.08.1 release 

Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

File: scripts/generate-changelog [hunk 1]
```diff
@@ -44,8 +44,8 @@ Github personal access token: https://github.com/settings/tokens, and make you h
 Usage:
     cd spark-rapids/
 
-    # generate changelog for release 0.1,0.2,0.3,0.4,0.4.1,0.5,21.06,21.06.1,21.06.2,21.08
-    scripts/generate-changelog --token=<GITHUB_PERSONAL_ACCESS_TOKEN> --releases=0.1,0.2,0.3,0.4,0.4.1,0.5,21.06,21.06.1,21.06.2,21.08
+    # generate changelog for release 0.1,0.2,0.3,0.4,0.4.1,0.5,21.06,21.06.1,21.06.2,21.08,21.08.1
+    scripts/generate-changelog --token=<GITHUB_PERSONAL_ACCESS_TOKEN> --releases=0.1,0.2,0.3,0.4,0.4.1,0.5,21.06,21.06.1,21.06.2,21.08,21.08.1
 
     # generate changelog for release 0.1,0.2,0.3,0.4,0.4.1,0.5,21.06,21.06.1,21.06.2,21.08 to /tmp/CHANGELOG.md
     GITHUB_TOKEN=<GITHUB_PERSONAL_ACCESS_TOKEN> scripts/generate-changelog --releases=0.1,0.2,0.3,0.4,0.4.1,0.5,21.06,21.06.1,21.06.2,21.08 --path=/tmp/CHANGELOG.md
```
Comment (pxLi) [inline scripts/generate-changelog:51]: can you also update comment for these 2 lines?

Comment (NvTimLiu) [reply scripts/generate-changelog]: fixed

**Review by pxLi — APPROVED** [review]

**Comment by NvTimLiu** [comment]
build

**Pull request merged**